packages/build-utils/src/edge-function.ts

```diff
@@ -44,6 +44,12 @@ export class EdgeFunction {
   /** Cronjob definition for the edge function */
   cron?: Cron;
 
+  /** The framework */
```

Comment:
```suggestion
  /** The framework which created this function */
```

Reply:
Actually, can we stick this in `types.ts` so that we can share the same interface (and doc comment) with both `EdgeFunction` and `Lambda`.

Reply:
`Framework` or `FunctionFramework`?

Reply:
Let's go with `FunctionFramework` since there's already a `Framework` type in `@vercel/frameworks`.